src/parser.coffee

```diff
@@ -264,7 +269,7 @@ parseListItem = (argumentString) ->
 
   if nameMatches = new RegExp(ArgumentListItemRegex).exec(argumentString)
     name = nameMatches[1]
-    description = description.replace(nameMatches[0], '')
+    description = description.replace(nameMatches[0], '') if nameMatches[0]?
```

Comment:
I'm pretty sure that the 0th match can never be undefined?  If there's no match `nameMatches` itself would be undefined.

Reply:
Good point, I was just looking at anything that was <matchName>[index], will fix.